templates/dex.yaml

```diff
@@ -65,6 +65,11 @@ spec:
             # This URI is just a placeholder and it will be replaced during the
             # deployment with URL pointing to the traefik ingress public
             # load balancer.
-            - 'https://dex-k8s-authenticator-kubeaddons.kubeaddons.svc.cluster.local:5555/dex-k8s-authenticator/callback/kubernetes-cluster'
+            - 'https://PUBLIC.URI/dex-k8s-authenticator/callback/kubernetes-cluster'
           name: 'Kubernetes CLI authenticator'
           # secret: value is generated automatically - see `setupDex()`
+        - id: traefik-forward-auth
+          redirectURIs:
+            - 'https://PUBLIC.URI/_oauth'
+          name: 'Ops Portal authenticator'
+          # secret: ZXhhbXBsZS1hcHAtc2VjcmV0
```

Comment:
umm, I hope this isn't the value of the secret :). If we don't need it we could remove it.

Reply:
All secrets get generated in `kubeaddons`. I will clean this up.